README.md

```diff
@@ -1,485 +1,7 @@
-
-The WebSocket server doesn't have access to the session, but it has access to the cookies. This can be used when you need to handle authentication. You can see one way of doing that with Devise in this [article](http://www.rubytutorial.io/actioncable-devise-authentication).
-
-## Dependencies
-
-Action Cable is currently tied to Redis through its use of the pubsub feature to route
-messages back and forth over the WebSocket cable connection. This dependency may well
-be alleviated in the future, but for the moment that's what it is. So be sure to have
-Redis installed and running.
-
-The Ruby side of things is built on top of [faye-websocket](https://github.com/faye/faye-websocket-ruby) and [celluloid](https://github.com/celluloid/celluloid).
-
-
-## Deployment
-
-Action Cable is powered by a combination of EventMachine and threads. The
-framework plumbing needed for connection handling is handled in the
-EventMachine loop, but the actual channel, user-specified, work is handled
-in a normal Ruby thread. This means you can use all your regular Rails models
-with no problem, as long as you haven't committed any thread-safety sins.
-
-But this also means that Action Cable needs to run in its own server process.
-So you'll have one set of server processes for your normal web work, and another
-set of server processes for the Action Cable. The former can be single-threaded,
-like Unicorn, but the latter must be multi-threaded, like Puma.
-
-
-## Alpha disclaimer
-
-Action Cable is currently considered alpha software. The API is almost guaranteed to change between
-now and its first production release as part of Rails 5.0. Real applications using the framework
-are all well underway, but as of July 8th, 2015, there are no deployments in the wild yet.
-
-So this current release, which resides in rails/actioncable, is primarily intended for
-the adventurous kind, who do not mind reading the full source code of the framework. And it
-serves as an invitation for all those crafty folks to contribute to and test what we have so far,
-in advance of that general production release.
-
-Action Cable will move from rails/actioncable to rails/rails and become a full-fledged default
-framework alongside Action Pack, Active Record, and the like once we cross the bridge from alpha
-to beta software (which will happen once the API and missing pieces have solidified).
-
-Finally, note that testing is a unfinished/unstarted area of this framework. The framework
-has been developed in-app up until this point. We need to find a good way to allow the user to test
-their connection and channel logic.
-
-
-## License
-
-Action Cable is released under the MIT license:
-
-* http://www.opensource.org/licenses/MIT
-
-
-## Support
-
-Bug reports can be filed for the alpha development project here:
-
-* https://github.com/rails/actioncable/issues
+An archive of the source code before the merge is available [here](../../tree/archive).
```

Comment:
Actually the link is correct… I was just mimicking https://github.com/rails/activejob/pull/109

Just like @chancancode suggested (and @rafaelfranca did) in that situation, **You should branch master into a new branch called "archive" before doing this**, and should probably disable the issue tracker as well.